system/lib/sbrk.c

```diff
@@ -27,30 +27,57 @@ extern size_t emscripten_get_heap_size(void);
 #endif
 
 void *sbrk(intptr_t increment) {
-  intptr_t* sbrk_ptr = emscripten_get_sbrk_ptr();
-  intptr_t old_brk = *sbrk_ptr;
-  // TODO: overflow checks
-  intptr_t new_brk = old_brk + increment;
-#if __wasm__
-  uintptr_t old_size = __builtin_wasm_memory_size(0) * WASM_PAGE_SIZE;
+#if __EMSCRIPTEN_PTHREADS__
```

Comment:
existing code in emscripten seems inconsistent about whether it should be this or `#ifdef __EMSCRIPTEN_PTHREADS__` :woman_shrugging:

Reply:
Yeah, I copied this from dlmalloc, but I'm not surprised other code does differently... I don't feel strongly either way.